### PR TITLE
Stormpath is closing, joined forces with okta.com

### PR DESCRIPTION
**Pull request by Morrolan**
Stormpath is closing it's API on 17th August 2017 as it has joined forces with okta.dom - replaced Stormpath entry with Okta

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -525,3 +525,4 @@ API | Description | Auth | HTTPS | Link |
 | Weatherbit | Weather | `apiKey` | Yes | [Go!](https://www.weatherbit.io/api) |
 | Wunderground | Weather | `apiKey` | Yes | [Go!](https://www.wunderground.com/weather/api/) |
 | Yahoo! Weather | Weather | No | Yes | [Go!](https://developer.yahoo.com/weather/) |
+okta
```
Comment (jbrooksuk) [inline README.md]: Whoops! This needs removing.

**Comment by Morrolan** [comment]
Not sure if I've reviewed the changes correctly but the file now appears correct to me?

**Comment by davemachado** [comment]
Hi @Morrolan , thanks for the PR! The JSON files for this project are auto-generated, so could you please remove them from your submission? Thanks!

**Comment by mikestreety** [comment]
@Morrolan nudge :)

**Comment by Morrolan** [comment]
Sorry for the delay - I'm not sure how to actually remove the specified file from the pull request?

**Comment by davemachado** [comment]
@Morrolan all you need to do is make a new commit on this branch that deletes the file and then push it up.

**Comment by davemachado** [comment]
Thanks @Morrolan !

**Pull request merged**